ansible/ACI/aci_deploy.yml

```diff
@@ -0,0 +1,78 @@
+# Description
+# ===========
+# This playbook build the sample application into a docker image, push the image into ACR and create ACI from the image.
```

Comment:
do you want to include dependencies in comments to let user running docker before run this playbook

Reply:
Added.